### DISCUSSION TITLE
[Help Wanted] Bulk migrate from Joplin?

### DISCUSSION THREAD

**Discussion by mtutty** [standardnotes/forum#2736] [Suggestions]
I've spent some time googling and reading, but I don't see an authoritative guide for migrating dozens or hundreds of notes and tags from Joplin.  Can someone steer me in the right direction?

**Comment by JaspalSuri** [comment]
Hi @mtutty,

While there isn't dedicated support for importing notes created in Joplin, you can use either the Markdown or Joplin Export Directory options (from Joplin's desktop client) and then use [plaintext file converter](https://dashboard.standardnotes.org/tools) to convert the files into a form that Standard Notes can import. 

Tags would have to be manually added to each note. Images would have to be uploaded via the FileSafe extension and can be viewed within the Bold Editor (which renders HTML); both are available through our [Extended](https://standardnotes.org/extensions) plan. We plan to provide ways to natively upload and view images from within our apps in future updates. 

I hope this helps; please let me know if you have any additional questions!
